Homework_2/task_2.py

```diff
@@ -0,0 +1,46 @@
+#Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. 
+#Программа должна возвращать сумму и произведение* дробей. 
+#Для проверки своего кода используйте модуль fractions.
+
+from fractions import Fraction as frac
+
+n, f = input('Введите первую дробь: '), input('Введите вторую дробь: ')
+index = 0
+a_string, b_string, c_string, d_string = '', '', '', ''
+while (n[index] != '/'):
+    a_string = a_string + n[index]
+    index += 1
+
+a = int(a_string)
+index += 1
+
+while (index < len(n)):
+    b_string = b_string + n[index]
+    index += 1
+
+b = int(b_string)
+
+index = 0
+
+while (f[index] != '/'):
+    c_string = c_string + f[index]
+    index += 1
+
+c = int(c_string)
+index += 1
+
+while (index < len(f)):
```

Comment:
Что что там много циклов `while`
```suggestion
while index < len(f):
```